src/model/src/ASX_DATA_MOD.F

```diff
@@ -941,36 +897,28 @@ Subroutine GET_MET ( JDATE, JTIME, TSTEP )
 
          call interpolate_var ('QC', jdate, jtime, Met_Data%QC)
 
+          call interpolate_var ('UWINDA', jdate, jtime, Met_Data%UWINDA)
+
+          call interpolate_var ('VWINDA', jdate, jtime, Met_Data%VWINDA)
 C-------------------------------- MET_CRO_2D --------------------------------
 C Vegetation and surface vars
+
          call interpolate_var ('LAI', jdate, jtime, Met_Data%LAI)
 
          call interpolate_var ('VEG', jdate, jtime, Met_Data%VEG)
 
          call interpolate_var ('ZRUF', jdate, jtime, Met_Data%Z0)
 
-C Canopy vars (Wei Li)
```

Comment:
Please revert to include all my canopy changes here in updated ASX_DATA_MOD.

Reply:
I have made updates, move back all canopy inline for ASX_DATA_MOD